packages/fxa-auth-server/lib/senders/emails/styles.ts

```diff
@@ -0,0 +1,61 @@
+export const style = `
+.header-text div{
+    font-family: sans-serif !important;
+    font-size: 21px !important;
+    line-height: 29px !important;
+    font-weight: normal !important;
+    text-align: center !important;
+    }
+
+td {
+    padding: 5px !important;
+    }
+
+span {
+    color: #0a84ff;
+    }
+
+.primary-text div {
+    font-family: sans-serif !important;
+    font-size: 14px !important;
+    line-height: 21px !important; 
+    font-weight: normal !important; 
+    }
+
+.secondary-text div {
+    font-family: sans-serif !important;
+    font-weight: normal !important;
+    text-align: center !important;
+    color: #737373 !important;
+    font-size: 11px !important;
+    line-height: 18px !important;
+    }
+
+.app-badges img {
```

Comment:
I'm not sure these should go in a "global" styles file since not all of them will reference `app-badges`, this one is going to get inserted on every template right? Also OK for that to happen in a follow up once the CSS approach is better sussed out... it'd be nice if we could use SCSS in this kind of file.